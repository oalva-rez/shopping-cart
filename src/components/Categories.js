import React from "react";
import men from "../assets/men.png";
import women from "../assets/women.png";
import athletics from "../assets/athletics.png";

export default function Categories() {
  return (
    <section className="categories">
      <div>
        <img src={men} alt="men" />
        <h3>Men</h3>
      </div>
      <div>
        <img src={women} alt="women" />
        <h3>Women</h3>
      </div>
      <div>
        <img src={athletics} alt="athletics" />
        <h3>Athletics</h3>
      </div>
    </section>
  );
}
