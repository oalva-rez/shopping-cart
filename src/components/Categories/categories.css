.categories {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  align-items: center;
  margin: 2rem auto;
  padding: 0;
  width: 100%;
  max-width: 1700px;
  gap: 2rem;
}
.categories > * {
  position: relative;
}
.categories h3 {
  position: absolute;
  font-weight: normal;
  top: 70%;
  left: 20%;
  transform: translate(-50%, -50%);
  color: black;
  font-size: 2rem;
}
