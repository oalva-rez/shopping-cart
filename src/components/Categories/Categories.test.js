import { render, screen } from "@testing-library/react";
import Categories from "./Categories";
import "@testing-library/jest-dom";

describe("Categories", () => {
  it("matches snapshot", () => {
    const { container } = render(<Categories />);
    expect(container).toMatchSnapshot();
  });
});
