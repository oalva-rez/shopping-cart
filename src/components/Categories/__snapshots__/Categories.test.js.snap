// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Categories matches snapshot 1`] = `
<div>
  <section
    class="categories"
  >
    <div>
      <img
        alt="men"
        src="men.png"
      />
      <h3>
        Men
      </h3>
    </div>
    <div>
      <img
        alt="women"
        src="women.png"
      />
      <h3>
        Women
      </h3>
    </div>
    <div>
      <img
        alt="athletics"
        src="athletics.png"
      />
      <h3>
        Athletics
      </h3>
    </div>
  </section>
</div>
`;
